### DISCUSSION TITLE
Job system

### DISCUSSION THREAD

**Discussion by TheAtomAnt** [Eldemarkki/Marching-Cubes-Terrain#50] [Ideas]
I noticed that the job system wasn't really being used to it's fullest. I forked this repo and made some modifications. Take a look, perhaps you could find use of those changes.

**Comment by Eldemarkki** [comment]
Hey, thank you! That's a great contribution! I merged it into the develop branch from your repo. 

For some reason, your profile does not show up on the [contributions page](https://github.com/Eldemarkki/Marching-Cubes-Terrain/graphs/contributors), but maybe it only shows people who have contributed to the master branch. So when I eventually merge develop into master, it should show up there finally. I'm not quite sure how the contributors and pull requests from other peoples' repositories work, but if it doesn't show up then I'll try to figure out another way to credit you. 

Still, thank you!
